asps/Simulation/starsim/agzio/agzkine.age

```diff
@@ -110,8 +110,13 @@
       If (IKINEold==-2) call AgNTread(ier)
       If (IKINEold<=-3) call AgFread (ier)
       If (IKINEold<=-4)
-      { J=CsADDR('AGUSREAD'); If (J>0) Call CSjCAL(J,1,Ier,0,0,0,0,0,0,0,0,0) }
+      { 
 
+        J             =   CsADDR('AGUSREAD')
+        IF (J <= 0) J =   CsADDR('AGEVENTREAD')
+        If (J>0) Call CSjCAL(J,1,Ier,0,0,0,0,0,0,0,0,0)
```

Comment:
`J <= 0` and `J>0` will never coincide. Why set J at all if it's never gonna be used?

If it is used, it would help to swap the statements and add a comment
```suggestion
        If (J>0) Call CSjCAL(J,1,Ier,0,0,0,0,0,0,0,0,0)
c       Set J to be used blablabla
        IF (J <= 0) J =   CsADDR('AGEVENTREAD')
```

Reply:
I now remembered. Reading this last time I understood correctly. `J <= 0` means lookup fail, so we try `AGEVENTREAD`. Never mind then.